packages/clerk-js/src/utils/zxcvbn.ts

```diff
@@ -2,33 +2,32 @@ import type { ZxcvbnResult } from '@clerk/types';
 
 export type zxcvbnFN = (password: string, userInputs?: (string | number)[]) => ZxcvbnResult;
 
-export const loadZxcvbn = () => {
-  return loadLib().then(([coreModule, languageCommonModule]) => {
-    const { zxcvbnOptions, zxcvbn } = coreModule;
-    const { dictionary, adjacencyGraphs } = languageCommonModule;
-    zxcvbnOptions.setOptions({
-      dictionary: {
-        ...dictionary,
-      },
-      graphs: adjacencyGraphs,
-    });
-    return zxcvbn;
+export const loadZxcvbn = async () => {
+  const { core, languageCommon } = await loadLib();
+  const { zxcvbnOptions, zxcvbn } = core;
+  const { dictionary, adjacencyGraphs } = languageCommon;
+  zxcvbnOptions.setOptions({
+    dictionary: {
+      ...dictionary,
+    },
+    graphs: adjacencyGraphs,
   });
+  return zxcvbn;
 };
 
 // We can only dynamically import zxcvbn in the browser.
 // On the headless variant we need to require it, and
 // this is because import() uses the document object
 // which is not available in Expo or other native environments.
-const loadLib = () => {
+const loadLib = async () => {
   let core;
   let languageCommon;
   if (__IS_NATIVE__) {
-    core = require('@zxcvbn-ts/core');
-    languageCommon = require('@zxcvbn-ts/language-common');
+    core = await require('@zxcvbn-ts/core');
+    languageCommon = await require('@zxcvbn-ts/language-common');
```

Comment:
Isn't require synchronous ?

Reply:
@anagstef I also think that the `await` before the require isn't doing anything 👍

Reply:
Yes, I've removed this, but haven't pushed yet! Thank you both for noticing!